packetery/src/Packetery/Checkout.php

```diff
@@ -114,25 +114,21 @@ public function __construct( Engine $latte_engine, Provider $options_provider, R
 	/**
 	 * Checks if chosen carrier has pickup points and sets carrier id in provided array.
 	 *
-	 * @param string $chosenMethod Shipping rate id.
-	 * @param array  $matches Array with carrier id.
+	 * @param string|null $carrierId Carrier id.
 	 *
 	 * @return bool
 	 */
-	public function isPickupPointMethod( string $chosenMethod, array &$matches ): bool {
-		if ( strpos( $chosenMethod, 'zpoint' ) !== false ) {
-			$matches[1] = 'packeta';
-
-			return true;
+	public function isPickupPointCarrier( ?string $carrierId ): bool {
+		if ( null === $carrierId ) {
+			return false;
 		}
-
-		if ( preg_match( '/^' . self::CARRIER_PREFIX . '(\d+)$/', $chosenMethod, $matches ) ) {
-			$isPickupPoints = $this->carrierRepository->getIsPickupPoints( (int) $matches[1] );
-
-			return ( '1' === $isPickupPoints );
+		// TODO: use constant Repository::INTERNAL_PICKUP_POINTS_ID.
```

Comment:
proč toto neuděláme hned ? Co nám v tom brání ?

Reply:
Protože už je to připraveno v https://github.com/Zasilkovna/WooCommerce/pull/38